middle_end/flambda2/from_lambda/lambda_to_flambda.ml

```diff
@@ -1662,7 +1662,7 @@ and cps_function env ~fid ~(recursive : Recursive.t) ?precomputed_free_idents
   in
   let params =
     List.map
-      (fun (param, kind) -> param, Flambda_kind.With_subkind.from_lambda kind)
+      (fun (param, kind, _attr) -> param, Flambda_kind.With_subkind.from_lambda kind)
```

Comment:
same here: maybe at least have a `Let No_attributes = attr in ...` to help when adding new attributes